tests/scripts/pulpcore/test_pulpimporter.sh

```diff
@@ -5,6 +5,8 @@
 
 pulp debug has-plugin --name "file" || exit 23
 
+eval "$(pulp status | jq -r '.domain_enabled // false | not')" || exit 23
```

Comment:
We could have a `pulp debug` command like `pulp debug has-plugin`.

Reply:
> We could have a `pulp debug` command like `pulp debug has-plugin`.

@mdellweg  Open an issue pliz!